src/pages/resultsView/ResultsViewURLWrapper.ts

```diff
@@ -0,0 +1,66 @@
+import URLWrapper from "../../shared/lib/URLWrapper";
+import ExtendedRouterStore from "../../shared/lib/ExtendedRouterStore";
+import {computed} from "mobx";
+import autobind from "autobind-decorator";
+import {ResultsViewTab} from "pages/resultsView/ResultsViewPageHelpers";
+
+export type ResultsViewURLQuery = {
+    clinicallist:string;
+    gene_list:string;
+    cancer_study_list:string;
+    case_ids:string;
+    sample_list_ids:string;
+    case_set_id:string;
+    profileFilter:string;
+    RPPA_SCORE_THRESHOLD:string;
+    Z_SCORE_THRESHOLD:string;
+    geneset_list:string;
+    treatment_list: string;
+
+    genetic_profile_ids_PROFILE_MUTATION_EXTENDED:string;
+    genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION:string;
+    genetic_profile_ids_PROFILE_MRNA_EXPRESSION:string;
+    genetic_profile_ids_PROFILE_PROTEIN_EXPRESSION:string;
+    genetic_profile_ids_PROFILE_GENESET_SCORE:string;
+    genetic_profile_ids_GENERIC_ASSAY:string;
+    genetic_profile_ids:string;
+
+};
+
+export default class ResultsViewURLWrapper extends URLWrapper<ResultsViewURLQuery> {
+    constructor(routing:ExtendedRouterStore) {
+        super(routing, [
+            { name:"clinicallist", isSessionProp:false },
+            { name: "gene_list", isSessionProp:true },
```

Comment:
can you space these out so that the isSessionProp:false is spaced from the others, its hard to spot otherwise